### ISSUE TITLE
Collection is not visible if two account are logged in

### ISSUE THREAD

**Issue by calebdias** [bitwarden/android#3768]
### Bitwarden Beta

- [X] I'm using the new native Bitwarden Beta app and I'm aware that legacy .NET app bugs should be reported in [bitwarden/mobile](https://github.com/bitwarden/mobile)

### Steps To Reproduce

1. Log in with a BW account (account "A") that has access to an Organization's Collection.
2. From the same app, log in to a different BW account (account "B") that has access to the same Collection.
3. Now switch back to account "A". The collection is no longer visible.

### Expected Result

Ability to sign in with two BW accounts and access a shared Collection from each.  (This works as expected in the legacy app).

### Actual Result

When I log in with two BW accounts that have access to the same Collection, only the second BW account can see the Collection.

### Additional Context

Upon initial login, account "A" has access to the shared Collection as expected.  Only once account"B" logs in, does account "A" lose access to the Collection.

### Build Version

2024.7.3 (18938)

### Environment Details

Google Pixel 7a, GrapheneOS.

### Issue Tracking Info

- [X] I understand that work is tracked outside of Github. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by sammbw** [comment]
Hi there,

I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below. 

Thanks!

**Comment by starktower007** [comment]
I could reproduce this issue and it does indeed behave as @calebdias pointed out.

I also noticed that even logging out account "B" doesn't show the collection back in account "A".

The only ways to show the collection again in account A are

- Log out account A and log in back again. This shows the collection back in account A in the app. This also stops the collection from showing up in account B in the process.
- Create and trash a new login in the collection from account A on another device. This shows the collection back in account A in the app. This also stops the collection from showing up in account B in the process.

If you log out and log in from account "B" again, the collection again shows up only in account B.

**Comment by calebdias** [comment]
> Hi there,
> 
> I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below.
> 
> Thanks!

Thank you for looking into this.  Let me know if there are any specific questions I can answer, or ways I can help with further testing.

**Comment by justrightlizard** [comment]
I can reproduce the issue using the exact same steps from @calebdias's initial post and @starktower007's comment.

Here are a few notes about my situation:
1. Using version 2024.9.1 (19099)
2. Account A is a premium user. 
3. Account B is a free user.
4. Using the free tier of organizations.
5. Two collections created in total.
6. Account A has the Owner role
7. Account B has the User role.
8. Account A has the "can manage" permission for both collections. 
9. Account B has the "can edit" permission for both collections

I'm not sure how much of that matters, but I figured I'd add as much detail about the situation as possible.

**Comment by calebdias** [comment]
Any plans to fix this? This bug still affects the latest stable version.

**Comment by justrightlizard** [comment]
> Any plans to fix this? This bug still affects the latest stable version.

Yep, it still affects the latest pre-release as well. 2024.12.0 (19597)

Would love to see this fixed. Currently, I have to have both the pre-release and pre-release beta version installed on my phone to fulfill my use case.

**Comment by starktower007** [comment]
It seems like they are unable to reproduce many of the issues we report here and often they just leave them as it is after the usual "I am unable to reproduce this issue, it has been escalated for further investigation. If you have more information that can help us, please add it below" comment.  

https://github.com/bitwarden/android/issues/4459

 Can @calebdias @justrightlizard see if this issue can be reproduced?

**Comment by calebdias** [comment]
I found a workaround... Clicking the menu, then "sync" (not dragging down to refresh) will make the collection reappear.  

You will have to do this again if you want the collection go be visible from another account.

**Comment by pquantin** [comment]
The issue (and proposed workaround) is still present in 2025.3.0 version.

**Comment by closebot-bw** [comment]
⚠️ **Stale Issue Notice**

This issue has been automatically marked as stale due to inactivity. It will be closed in **2 weeks** (February 25, 2026) if no further activity occurs.

If this issue is still relevant and you would like to keep it open, please:
- Comment on this issue to show continued interest
- Provide any additional information or updates
- Confirm that the issue still exists in the latest version

Thank you for your contribution to this project! 🙏

**Comment by closebot-bw** [comment]
🔔 **Final Notice - Issue Will Be Closed Soon**

This issue was previously marked as stale and will be automatically closed in **2 days** (February 25, 2026) if no further activity occurs.

If you're still experiencing this issue or believe it should remain open, please comment below to prevent automatic closure.

We appreciate your understanding and contribution to keeping our issue tracker organized! 📋

**Comment by closebot-bw** [comment]
🔒 **Issue Closed Due to Inactivity**

This issue has been automatically closed due to lack of activity for an extended period. We periodically review and close inactive issues to help maintain our issue tracker and focus on current priorities.

**If this issue is still relevant:**
- Please create a new issue with updated information
- Include steps to reproduce the problem if it's a bug report
- Mention if this issue still occurs in the latest version

Thank you for your contribution to this project. Your feedback helps us improve! 🙏

**Comment by calebdias** [comment]
This is still relevant.
